XpressAI/xircuits
# added custom loading logo

# Description

We have a new custom Xpress logo now~

![Loading Screen mini](https://user-images.githubusercontent.com/68586800/155444149-555e2c24-cf05-4f3c-8e77-a95cfb55b0e9.gif)


## Pull Request Type


## Type of Change


# Tests

1. View New Loading Logo

    1. Launch Jupyterlab
    2. See logo
    3. Refresh Jupyterlab
    4. See logo again :D


## Tested on?


# Notes

Thanks media team.
